### ISSUE TITLE
lag when using mouse pad scroll the editor 

### ISSUE THREAD

**Issue by yinancx** [Microsoft/vscode#34372]
- VSCode Version: Code 1.16.0 (787b31c0474e6165390b5a5989c9619e3e16f953, 2017-09-06T16:27:49.891Z)
- OS Version: Windows_NT x64 10.0.16278
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|1.3.1
sublime-keybindings|ms-|2.9.1
vetur|oct|0.9.7
java|red|0.9.0
debug|web|0.21.2

(2 theme extensions excluded)

---

Steps to Reproduce:

1. use two fingers drag the mouse pad, quickly change the direction;
3. a several seconds delay to response

Reproduces without extensions: Yes

**Comment by alexandrudima** [comment]
@yinancx are you perhaps using a laptop which has a precision touch pad ?

**Comment by yinancx** [comment]
@alexandrudima I'm using Surface Pro 4 with TypeCover. No lag when scrolling by clicking and dragging the scrollbar slider with touch pad and NO LAG when using mouse

**Comment by MaoScut** [comment]
I'm using surface pro 4 too, and I have the same problem. And it is no lag when using mouse and touching the screen to scroll.

**Comment by JWMB** [comment]
SP4 pro here too, same problem, but not new - I've had it since I seriously started using vscode, probably since v1.15.

**Comment by alexandrudima** [comment]
I'm sorry, but in this case, it appears as a dup of #13612.

TL;DR: for whatever reason, it helps to resize VSCode and then re-maximize it and then scrolling with the precision touch pad is smooth again. My personal belief is that it is an Electron specific problem.
